platform-docs/src/components/HomepageTrustedBy/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import styles from './styles.module.css';
+
+const Users = [
+	{
+		title: 'Tumblr',
+		img: require( '@site/static/img/tumblr.png' ).default,
+	},
+	{
+		title: 'WordPress',
+		img: require( '@site/static/img/wordpress.png' ).default,
+	},
+	{
+		title: 'Day One',
+		img: require( '@site/static/img/dayone.png' ).default,
+	},
+];
```

Comment:
Happy to add more "Gutenberg" users, these are the ones I'm aware of.

Reply:
The [Drupal extension](https://www.drupal.org/project/gutenberg) might be worth a mention?

Reply:
I thought about it but at the moment the Drupal doesn't really use the block editor package as a framework but they reuse the whole plugin somehow. I know they're working on it to improve the integration so I left it out for now.